libcontainer/dmz/dmz_linux.go

```diff
@@ -1,4 +1,5 @@
-//go:build !runc_nodmz
+//go:build (386 || amd64 || arm || arm64 || loong64 || ppc64le || riscv64 || s390x) && !runc_nodmz
+// +build 386 amd64 arm arm64 loong64 ppc64le riscv64 s390x
```

Comment:
I don't think we need to add the arches here. IMHO, we should leave this as it was and just make all arches compile, as this PR is doing: https://github.com/opencontainers/runc/pull/4076/files